src/Components/Common/Button/Button.module.scss

```diff
@@ -0,0 +1,20 @@
+@import 'src/utils/styles/variables';
```

Comment:
As I see you inject the variable into each file manually =)
For future tickets try to insert it automatically, for that you may need to "eject react app" but please investigate it more =)

Reply:
Imported all variables and mixins into each scss-file automatically using sass-loader.